fe/fe-core/src/main/java/com/starrocks/statistic/FullStatisticsCollectJob.java

```diff
@@ -133,6 +134,8 @@ private Expr nowFn() {
     public void collectStatisticSync(String sql, ConnectContext context) throws Exception {
         LOG.debug("statistics collect sql : " + sql);
         StatisticExecutor executor = new StatisticExecutor();
+        SessionVariable sessionVariable = context.getSessionVariable();
```

Comment:
Would better comment why we need do disable page cache when  collect statistic.

Reply:
> Would better comment why we need do disable page cache when collect statistic.

done